geometry_msgs/Pose pose
std_msgs/Int64 id

std_msgs/UInt32 time_sec
std_msgs/UInt32 time_nsec

ndt_map/NDTMapMsg ndt_map #the map